ci-operator/jobs/openshift-qe-periodics.yaml

```diff
@@ -0,0 +1,61 @@
+periodics:
+- agent: kubernetes
+  decorate: true
+  interval: 24h
+  name: verification-tests-context
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/test-e2e-cluster-profile
+      - --target=test-e2e
+      - --template=/usr/local/test-e2e
+      - --git-ref=openshift/verification-tests@master
```

Comment:
This should be done with `extra_refs` on the job, not a flag

Reply:
`extra_refs` are not working :)